spec/std/io/multi_writer_spec.cr

```diff
@@ -49,4 +50,21 @@ describe "IO::MultiWriter" do
       io.closed?.should eq(true)
     end
   end
+
+  describe "#flush" do
+    it "writes to IO and File" do
+      with_tempfile("multiple_writer_spec") do |path|
+        file = File.new(path, "a")
```

Comment:
Are you using append mode on purpose? It probably doesn't matter be cause the temp path should not exist, but as far as I understand it, write mode would be more fitting.

Reply:
done